eland/operations.py

```diff
@@ -285,6 +287,45 @@ def _metric_aggs(
                             results[field] = response["aggregations"][
                                 "percentiles_" + field
                             ]["values"]["50.0"]
+                        elif func[1] == "variance":
+                            # pandas computes the sample variance
+                            # Elasticsearch computes the population variance
+                            count = response["aggregations"][func[0] + "_" + field][
+                                "count"
+                            ]
+
+                            results[field] = response["aggregations"][
+                                func[0] + "_" + field
+                            ][func[1]]
+
+                            # transform population variance into sample variance
+                            if count == 0:
+                                results[field] = pd.NaN
```

Comment:
Can we have test cases for var+std that hits the `pd.NaN` case

Reply:
++ - many thanks for spotting, `pd.NaN` is incorrect + test fails